ansible/roles/jenkins/tasks/main.yml

```diff
@@ -42,14 +42,11 @@
   shell: "{{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml --param BASE_IMAGE_TAG='nodejs:6' --param APP_BC='nodejs-mongo-persistent' --param APP_NAME='nodejs-mongo-persistent' --param EMAIL_LIST='aweiteka@redhat.com jcallen@redhat.com' --param=APP_PIPELINE='lifecycle-app-lifecycle' -n {{ project_name }} | {{ oc }} apply --force=true -f - -n {{ project_name }}"
 - name: Create Jenkins base image pipeline
   shell: "{{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml -n {{ project_name }} | {{ oc }} apply --force=true -f - -n {{ project_name }}"
-- name: Get last jenkins-custom build number
-  command: "{{ oc }} get bc jenkins-custom --template '{{ '{{' }} .status.lastVersion {{ '}}' }}'"
-  register: jenkins_custom_build_number
 - name: Get last jenkins-custom build status
-  command: "{{ oc }} get build jenkins-custom-{{ jenkins_custom_build_number.stdout }} --template '{{ '{{' }} .status.phase {{ '}}' }}'"
+  command: "{{ oc }} get build jenkins-custom-1 -n {{ project_name }} --template '{{ '{{' }} .status.phase {{ '}}' }}'"
```

Comment:
@codificat What if the build number is something other than one?

Reply:
Right, I was thinking only on initial run, when the buildConfig is created - at which point the first build will obviously be build 1... but certainly it won't be when this runs again.

On the other hand: what triggered this change was a failure of the playbook due to a race: reaching that check before any build was triggered - resulting in a value of `0` in the build number check.

Bottom line: this part needs some improvement, will re-review.

Reply:
FWIW, the goal is to be able to run ansible any time without arbitrarily starting builds.

I know what we had wasn't working every time. thanks for looking at it.